documentation/agent/installation/linux/deb.md

```diff
@@ -242,6 +242,14 @@ METHOD 3:
         <td>2.3.18</td>
         <td>Debian 9 minimal (stretch/testing)</td>
     </tr>
+     <tr>
+        <td>2021/06/08</td>
+        <td>2.6.1</td>
```

Comment:
2.6.1 is not an existing version, but 2.6-1 is as this means it is the revision 1 of the 2.6 debian package.